packages/client/src/grant-token.ts

```diff
@@ -38,3 +38,27 @@ export const grantTokenByAuthorizationCode = async (
 
   return result.data;
 };
+
+export const grantTokenByRefreshToken = async (
+  endpoint: string,
+  clientId: string,
+  refreshToken: string
```

Comment:
just curious without the PKCE validation, could it be malicious attack base a refresh_token grant flow?